### ISSUE TITLE
Initialization of PhysFS in ElementaryOS fails

### ISSUE THREAD

**Issue by darltrash** [ReFreezed/HotParticles#1]
Universal failed to run in linux.

- OS: ElementaryOS (Ubuntu 18.04)
- Arch: x86_64
- Love2D Version: 11.3 (Mysterious Mysteries)
- Release loaded: Latest universal

Log:
```
Running Hot Particles

Error: D:/Projects/Lua/Gloa/modules/love/physfs.gloa:41: Assertion failed!
stack traceback:
	[ C ]: 'error_F1684'
	D:/Projects/Lua/Gloa/modules/basic.gloa:37: 'assert' (main.lua:1651)
	D:/Projects/Lua/Gloa/modules/love/physfs.gloa:41: 'init' (main.lua:2158)
	D:/Projects/Lua/Gloa/modules/love/physfs.gloa:111: 'getSearchPaths' (main.lua:2174)
	src/main.gloa:805: (main.lua:661)
	[ C ]: 'xpcall'
	D:/Projects/Lua/Gloa/modules/utils.gloa:27: field 'load' (main.lua:26)
	src/main.gloa:732: (main.lua:610)
	[ C ]: 'xpcall'
	D:/Projects/Lua/Gloa/modules/utils.gloa:27: (main.lua:26)
	[ C ]: 'xpcall'
	[Lua]: (boot.lua:793)
	[ C ]: 'xpcall'
	[Lua]: (boot.lua:802)
```

**Comment by ReFreezed** [comment]
Can you confirm if this is fixed in [v1.1](https://github.com/ReFreezed/HotParticles/releases/tag/1.1.0)?

**Comment by darltrash** [comment]
> Can you confirm if this is fixed in [v1.1](https://github.com/ReFreezed/HotParticles/releases/tag/1.1.0)?

It's working now! Thank you!
